packages/data-stores/src/user/test/actions.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * These tests shouldn't require the jsdom environment,
+ * but we're waiting for this PR to merge:
+ * https://github.com/WordPress/gutenberg/pull/20486
+ *
+ * @jest-environment jsdom
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createActions } from '../actions';
+
+const client_id = 'magic_client_id';
+const client_secret = 'magic_client_secret';
+
+describe( 'createAccount', () => {
+	const { createAccount } = createActions( {
+		client_id,
+		client_secret,
+	} );
+
+	const params = {
+		email: 'test@example.com',
+		extra: {
+			username_hint: 'Test Site Title',
+		},
+	};
+
+	const defaults = {
+		client_id,
+		client_secret,
+		is_passwordless: true,
+		signup_flow_name: 'gutenboarding',
+		locale: 'en',
+	};
+
+	it( 'requests a new passwordless account to be created', () => {
+		const generator = createAccount( params );
+
+		expect( generator.next().value ).toEqual( { type: 'FETCH_NEW_USER' } );
+
+		const apiResponse = {
+			success: true,
+			bearer_token: 'bearer-token',
+			username: 'testusernamer12345',
+			user_id: 12345,
+		};
+
+		expect( generator.next().value ).toEqual( {
+			type: 'WPCOM_REQUEST',
+			request: expect.objectContaining( {
+				body: {
+					...defaults,
+					...params,
+					validate: false,
```

Comment:
Seems like `validate: false` belongs in `defaults` too

Reply:
The behaviour in the action at the moment is that whatever value is passed in is overridden with `validate: false`, so I'll leave this here for the moment, but see if it needs updating again when we look at adding in a `validateAccount` action. Semantically, you're right though, it makes more sense for it to be in `defaults`. (Maybe another variable called `overrides`?)